NeuroID/src/main/java/com/neuroid/tracker/NeuroIDPublic.kt

```diff
@@ -0,0 +1,189 @@
+package com.neuroid.tracker
+
+import android.app.Activity
+import androidx.annotation.VisibleForTesting
+import com.neuroid.tracker.models.SessionStartResult
+
+interface NeuroIDPublic {
+    /**
+     * Deprecated, do not use!
+     */
+    @Deprecated("Replaced with getUserID", ReplaceWith("getUserID()"))
```

Comment:
Should this point to `getClientID` instead or `getUserID`?

Reply:
updated